### DISCUSSION TITLE
Add scheduled siad restarts

### DISCUSSION THREAD

**Discussion by MeijeSibbel** [siacentral/host-manager#107] [Feature Requests]
It appears that restarting siad every once in a while solves the stale contracts problem. Although this is probably something that should be fixed on the Sia side, it might be worth adding the option to restart siad every 24h/1week etc.

**Comment by n8mgr** [comment]
I definitely agree, automatic restarts would be beneficial. There could be issues with Sia failing to start, it would still need to be checked.
